src/hero.py

```diff
@@ -0,0 +1,9 @@
+def hero(bullets, dragons):
+    if dragons * 2 <= bullets:
+        return True
+    else:
+        return False
```

Comment:
`dragons * 2 <= bullets` evaluates to either true or false so just return it like so:
```python
return dragons * 2 <= bullets
```

Reply:
Okey dokey, thank you for helping!